src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -117,6 +117,25 @@ export default class DefaultBrowserBehavior implements BrowserBehavior {
     return s.join(', ');
   }
 
+  async supportedVideoCodecs(): Promise<string[]> {
+    const pc = new RTCPeerConnection();
+    pc.addTransceiver('video', { direction: 'inactive', streams: [] });
+    return (await pc.createOffer({ offerToReceiveVideo: true })).sdp
+      .split('\r\n')
+      .filter(x => {
+        return x.includes('a=rtpmap:');
+      })
+      .map(x => {
+        return x.replace(/.* /, '').replace(/\/.*/, '');
+      })
+      .filter((v, i, a) => {
```

Comment:
Could you explain what this check is for?

Reply:
This filter dedupes the list.